src/pages/AuthPage/AuthPage.tsx

```diff
@@ -1,7 +1,75 @@
-import * as React from 'react';
+import { ApolloError, gql, useMutation } from '@apollo/client';
+import React, { FunctionComponent, useContext, useState } from 'react';
+import { RouteComponentProps } from 'react-router-dom';
```

Comment:
Проверь чтобы везде где ты импортишь интерфейс был import type

```suggestion
import type { RouteComponentProps } from 'react-router-dom';
```

Reply:
fix